automobile/car.md

```diff
@@ -120,18 +120,40 @@ where to put sensors (or actuators if needed), which are in the front, top, rear
 and center of the car. The position of the central sensors slot is always at 0 0
 0 (which is the center of the rear wheels axis). For the three other sensor
 slots, the positions are different for each model (because the size of the cars
-differs), see the [following table](#slotpositions) for the exact positions.
+differs), see the [following table](#positions-of-the-car-sensors-slots) for the exact positions.
 
-%figure "Positions of the sensors slots"
+%figure "Positions of the car sensors slots"
 
 | Model              | Front slot translation | Top slot translation | Rear slot translation |
 | ------------------ | ---------------------- | -------------------- | --------------------- |
-| BmwX5              | 0.0 0.45 3.850         | 0.0 1.45 1.000       | 0.0 0.3 -1.000        |
-| LincolnMKZ         | 0.0 0.142 3.944        | 0.0 1.16 1.110       | 0.0 0.33 -1.06        |
+| BmwX5              | 0.0 0.45 3.85          | 0.0 1.45 1.0         | 0.0 0.3 -1.0          |
+| LincolnMKZ         | 0.0 0.142 3.944        | 0.0 1.16 1.11        | 0.0 0.33 -1.06        |
 | RangeRoverSportSVR | 0.0 0.5 3.5            | 0.0 1.3 1.4          | 0.0 0.33 -1.06        |
 | CitroenCZero       | 0.0 0.05 3.075         | 0.0 1.35 1.075       | 0.0 0.3 -0.425        |
-| ToyotaPrius        | 0.0 0.40 3.635         | 0.0 1.30 1.100       | 0.0 0.3 -0.850        |
+| ToyotaPrius        | 0.0 0.40 3.635         | 0.0 1.30 1.1         | 0.0 0.3 -0.850        |
 
 %end
 
 In order to simplify some simulations, `Solid` based cars can be used from the [VehicleSimple](#vehicle-simple) PROTO.
+
+### Heavy-weights
+
+Just like the car models presented above, two generic heavy-weights PROTO inherit from the `car` PROTO: a bus and a truck. In the case of the truck, a trailer can be present or not. The latter is based on a `Slot` node which is the endPoint of a `HingeJoint` allowing the trailer to freely rotate around its attachment point.
```

Comment:
It is very good to mention the HingeJoint so that the user understand it can rotate, but I don't think we need to mention the Slot.